src/avvie.in

```diff
@@ -0,0 +1,38 @@
+#!@PYTHON@
+
+# avvie.in
+#
+# Copyright 2022 mattia
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import os
+import sys
+import signal
+import locale
+import gettext
+
+VERSION = '@VERSION@'
+pkgdatadir = '@pkgdatadir@'
+localedir = '@localedir@'
+
+sys.path.insert(1, pkgdatadir)
+signal.signal(signal.SIGINT, signal.SIG_DFL)
+locale.bindtextdomain('avvie', localedir)
+locale.textdomain('avvie')
+gettext.install('avvie', localedir)
```

Comment:
This right over here is responsible for binding `_` globally as a translation function.

Reply:
Right. I suppose my question is then, how to I build and run the project for testing? (Sorry, I've not used meson before.)